EduardM/HW6/task_6.2.py

```diff
@@ -1,7 +1,7 @@
-while True:
-    login = input("Enter your login: ")
-    if login == "First":
-        print(f"Welcome back, {login}.")
-        break
-    else:
-        continue
+while True:

```

Comment:
good job
another way

user_name = input('Hello, please input your Log in:')

while user_name != 'First':
    
    user_name = input('Error: wrong username, please try one more time. Username:')
    

else:
    print('Greeting. Access granted!!!', user_name)